ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-main.yaml

```diff
@@ -176,6 +176,21 @@ tests:
   steps:
     cluster_profile: vsphere
     workflow: openshift-e2e-vsphere-etcd-scaling
+- as: e2e-nutanix-ovn
+  cluster: build01
+  optional: true
+  run_if_changed: pkg/controllers/controlplanemachinesetgenerator/.*|.*nutanix.*
```

Comment:
This should be
```
  run_if_changed: pkg/machineproviders/providers/openshift/machine/v1beta1/.*|.*nutanix.*
```

Reply:
Nutanix specific changes didn't happen on that path in that repository, though. https://github.com/openshift/cluster-control-plane-machine-set-operator/pull/200/files